### DISCUSSION TITLE
使用C++推理多张图像时，连续的识别与中间有间隔的识别推理时间不一致

### DISCUSSION THREAD

**Discussion by NidhoggBo** [PaddlePaddle/PaddleOCR#12351] [Q&A]
![微信图片_20240227134954](https://github.com/PaddlePaddle/PaddleOCR/assets/91516567/e9a51e87-5eb4-4b6f-82d3-9541720e0f32)
![微信图片_20240227135003](https://github.com/PaddlePaddle/PaddleOCR/assets/91516567/14087e9f-1749-4ea3-9e74-35ac6a2e8d63)

**Comment by TingquanGao** [comment]
请补充一下paddle版本，运行环境信息。已经完整代码是否可以提供呢？

**Comment by NidhoggBo** [comment]
> 请补充一下paddle版本，运行环境信息。已经完整代码是否可以提供呢？

paddleocr 2.7 visual studio2022
运行环境：win11 cuda 11.6 cudnn 8.6
[paddleocr.txt](https://github.com/PaddlePaddle/PaddleOCR/files/14441938/paddleocr.txt)
71-76行
